projecttacoma/ecqm-visualizers
# MeasureReport Population Table

# Summary
Created a table for displaying the population results of a FHIR MeasureReport

## New Behavior
New Component!! This component can be tested using the example app and will eventually be exported for use in fqm-testify

## Code Changes
- New Component: `PopulationResultsViewer`
- New types file for created types
- New testing
- Updated example app to allow for ease of testing

# Testing Guidance
- Run `npm run check`
- Reach out to me on Slack for testing example index.ts file
- Start up the example app with `cd example` then `npm start`
- Ensure the results are as expected
- Change the passed in MeasureReport array to instead be an array of the following type: `{label: string, report: fhir4.MeasureReport}`
- Rerun example app and ensure that the Patients are correctly identified by their passed-in labels
